packages/client/src/index.ts

```diff
@@ -1,14 +1,67 @@
 import { Property, Signature } from '@0auth/message';
+import {
+  encryptMessage,
+  getDecryptedMessageFromLocalStorage,
+  getRawKeyFromLocalStorageIfPresentElseGenerate,
+  storeMessageToLocalStorage,
+} from './utils';
+
+export enum StorageType {
+  LocalStorage,
+  IndexedDB,
+  ChromeExtension,
+}
 
-// TODO: @ts-ignore should be removed, when below function is implemented.
-// @ts-ignore
 export async function registerInfo(
   url: string,
   properties: Property[]
+  // TODO: @ts-ignore should be removed, when below function is implemented.
+  // @ts-ignore
 ): Promise<Signature> {
   // TODO: Not implemented yet.
 }
 
-export function storeSignature(properties: Property[], sign: Signature) {
+export function storeSignature(
+  properties: Property[],
+  sign: Signature,
+  storage: StorageType,
+  password?: string
+): void {
   // TODO: Not implemented yet.
+  switch (storage) {
+    case StorageType.LocalStorage:
+      const key = getRawKeyFromLocalStorageIfPresentElseGenerate(password);
+      storeMessageToLocalStorage(
+        encryptMessage(JSON.stringify({ properties, sign }), key),
+      );
```

Comment:
After reading the key from localStorage, 
the user information and the server's signature are encrypted and stored in localStorage.